akka-cluster/src/multi-jvm/scala/akka/cluster/SunnyWeatherSpec.scala

```diff
@@ -14,35 +14,61 @@ import scala.collection.immutable.SortedSet
 import akka.actor.Props
 import akka.actor.Actor
 
-object SunnyWeatherMultiJvmSpec extends MultiNodeConfig {
+class SunnyWeatherMultiJvmSpec(artery: Boolean) extends MultiNodeConfig {
   val first = role("first")
   val second = role("second")
   val third = role("third")
   val fourth = role("fourth")
   val fifth = role("fifth")
 
+  val common =
+    """
+    akka {
+      actor.provider = akka.cluster.ClusterActorRefProvider
+      loggers = ["akka.testkit.TestEventListener"]
+      loglevel = INFO
+      remote.log-remote-lifecycle-events = off
+      cluster.failure-detector.monitored-by-nr-of-members = 3
+    }
+    """
+
+  val arteryConfig =
+    """
+    akka.remote.artery {
+      enabled = on
+      port = 0
```

Comment:
look at `MultiNodeSpec`, it sets `"akka.remote.netty.tcp.port" -> selfPort` (and `selfPort` is by default 0)
I think we should amend, `MultiNodeSpec` to do the same with `akka.remote.artery.port`, and then this is not needed here.